src/examples/resources/MyCommands_fr.properties

```diff
@@ -0,0 +1,35 @@
+ban.name=bannissement
+ban.description=Bannir quelqu'un
```

Comment:
Does java handle the `'` here correctly? The github styling is off from it

Reply:
yes